app/serializers/found_gym_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class FoundGymSerializer
```

Comment:
possible refactor could be to added to the GymSerializer as a custom method to return all of these attributes